### ISSUE TITLE
Sync local playlists to YT Music playlists

### ISSUE THREAD

**Issue by HENRYMARTIN5** [BobbyESP/InnerTune#2]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

You can currently sync playlists from YT Music locally, but I'd like to be able to do it the other way around too. There's already an issue for this on the main InnerTune repo, but it seems to be no longer maintained, so I'm making this issue here so that there's a chance someone sees it.

### Why do you want this feature?

I use YT Music on other platforms than just mobile, and I like being able to create one unified playlist that properly syncs between all platforms.

**Comment by BobbyESP** [comment]
Hi! I will try to implement this ;)

**Comment by HENRYMARTIN5** [comment]
Thank you! This is one of the features I most need out of this app.

**Comment by siggi1984** [comment]
> Hi! I will try to implement this ;)

Yes, that function would be cool. Can you ask about the intermediate status of the function?

**Comment by OmerTurann** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.
JonatanSalas, @bhavishy6, @gengteng, @cooperliu101, @saeed-raeisi, @PitCoder, @veronica-silva, @gustavomts, @sombochea, @svaussier, @mdmmn378, @znipps, @abhik-99, @chenjunwen186, @mehmeta, @Leo-Lionni, @filipeakafilipe, @luoq, @xdanielsb, @psinogas

**Comment by BobbyESP** [comment]
> > Hi! I will try to implement this ;)
> 
> Yes, that function would be cool. Can you ask about the intermediate status of the function?

Hi! Sorry for the late response. I've been busy (and I am still), but I think that some of these days I will have this feature finished. It is indeed something necessary.
